awx/main/models/notifications.py

```diff
@@ -313,6 +313,7 @@ def context_stub(cls):
                            'modified': datetime.datetime(2018, 12, 13, 6, 4, 0, 0, tzinfo=datetime.timezone.utc),
                            'name': 'Stub JobTemplate',
                            'playbook': 'ping.yml',
+                           'scm_branch': '',
```

Comment:
blank indicates default branch from the project, just want to be on the same page about that.